src/kOS.Safe/Communication/Connection.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using kOS.Safe.Encapsulation;
+using kOS.Safe.Encapsulation.Suffixes;
+
+namespace kOS.Safe.Communication
+{
+    [kOS.Safe.Utilities.KOSNomenclature("Connection")]
+    public abstract class Connection<S> : Structure where S : SharedObjects
```

Comment:
We have 2 classes named `SharedObjects` one in kOS and one in kOS.Safe (the former inherits from the latter). `Connection` uses kOS.Safe.SharedObjects as a base class for the parameter. This allows us to keep it in kOS.Safe. Granted, currently this doesn't give us much, but I was thinking about writing unit tests that might involve this class some time in the future. Anyway, the more stuff is in kOS.Safe the better, right?